sqldelight-gradle/android-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightAndroidPlugin.kt

```diff
@@ -20,6 +20,8 @@ import com.android.build.gradle.AppPlugin
 import com.android.build.gradle.LibraryExtension
 import com.android.build.gradle.LibraryPlugin
 import com.android.build.gradle.api.BaseVariant
+import com.android.build.gradle.internal.errors.SyncIssueHandlerImpl
```

Comment:
This has the potential to likely break again in the future

Reply:
yes. Im hoping AGP3.2 will ship soonish since id rather wait for that before releasing 1.0